gsy_e_sdk/cli.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-arguments
```

Comment:
Nit: I think it would be better to place these pylint comments below the top-level docstring (above imports). It would make it easier to notice them.